eclair-core/src/main/scala/fr/acinq/eclair/channel/Helpers.scala

```diff
@@ -650,7 +654,7 @@ object Helpers {
       *
       * @return a [[RevokedCommitPublished]] object containing penalty transactions if the tx is a revoked commitment
       */
-    def claimRevokedRemoteCommitTxOutputs(keyManager: KeyManager, commitments: Commitments, tx: Transaction, db: ChannelsDb)(implicit log: LoggingAdapter): Option[RevokedCommitPublished] = {
+    def claimRevokedRemoteCommitTxOutputs(feeTargets: FeeTargets, keyManager: KeyManager, commitments: Commitments, tx: Transaction, db: ChannelsDb, feeEstimator: FeeEstimator)(implicit log: LoggingAdapter): Option[RevokedCommitPublished] = {
```

Comment:
`feeTargets` should be after `feeEstimator`

Reply:
Also done in f0649dd